src/NuGet.Core/NuGet.Protocol/Model/LocalPackageSearchMetadata.cs

```diff
@@ -145,5 +147,28 @@ public string LoadFileAsText(string path)
             }
             return fileContent;
         }
+
+        /// <summary>
+        /// Points to an Icon, either Embedded Icon or IconUrl
+        /// </summary>
+        public Uri GetIcon()
+        {
+            string embeddedIconPath = _nuspec.GetIcon();
```

Comment:
Misnamed variable which is an icon, not an iconPath.

Reply:
Renamed